setup.py

```diff
@@ -88,6 +88,7 @@ def get_packages(package):
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
```suggestion
```
we can't add the trove until https://github.com/aaugustin/websockets/issues/1000 is on pypi

Reply:
But they have added as well https://github.com/aaugustin/websockets/blob/6edb363af07365867b4c372405b6ab3177a57830/setup.py#L53

Reply:
Ah, it's on master.

Reply:
But `websockets` is not a mandatory dependency, so it shouldn't be a blocker, right?